rs/state_manager/src/manifest/split/tests.rs

```diff
@@ -397,24 +397,25 @@ fn expected_split_marker() -> (FileInfo, ChunkInfo) {
 /// `SUBNET_1`.
 ///
 /// `SystemMetadata` encodes the `CURRENT_CERTIFICATION_VERSION`, therefore the hashes in this
-/// test must be updated every time the current certification version is bumped.
+/// test must be updated every time the current certification version is bumped or a new
+/// field is added to `SystemMetadata`.
```

Comment:
It is only if there is a field that is added to system metadata that shows up in the serialization of a default system metadata that is constructed via `new`, right?

Reply:
I'm fine either way. I was mostly curious.